shared/styles/mixins.scss

```diff
@@ -83,17 +97,69 @@
   padding: 0 $container-gutter-mobile;
   max-width: $page-width + $container-gutter-mobile * 1;
 
-  @media (min-width: 340px) {
+  @media (min-width: $min-mobile) {
     max-width: $page-width + $container-gutter-mobile * 2;
   }
 
   @media (min-width: $min-tablet) {
-    padding: 0 $container-gutter-tablet;
+    padding-left: $container-gutter-tablet;
+    padding-right: $container-gutter-tablet;
     max-width: $page-width + $container-gutter-tablet * 2;
   }
 
   @media (min-width: $min-desktop) {
-    padding: 0 $container-gutter-desktop;
+    padding-left: $container-gutter-desktop;
+    padding-right: $container-gutter-desktop;
     max-width: $page-width + $container-gutter-desktop * 2;
   }
 }
+
+// change below to reflect your project
+// numerical values are arbitrary
+@mixin heading($min: ($font-size * 1.5), $max: ($font-size * 3)) {
+  @include reset-heading;
+  @include responsive-font($min, $max);
+
+  @warn 'Please implement for project';
+
+  margin-bottom: (5/$max)*1em;
+
+  font-family: $font-family;
+  font-weight: normal;
+  letter-spacing: 0;
+  line-height: (58/$max);
+
+  color: $color-font;
+}
+
+@mixin subheading($min: ($font-size * 1.5), $max: ($font-size * 3)) {
+  @include reset-heading;
+  @include responsive-font($min, $max);
+
+  @warn 'Please implement for project';
```

Comment:
What does `implement for project` mean?

Reply:
These three mixins are included for those that want them. They have some settings that need to be changed for the project you're using them for, so this should trigger you to change them. But the wording is bad.

`Please set values in this mixin for your project.`?

Reply:
Hmmm, are these really that different than any other piece of the starter kit? Anything _could_ be customized for a project if it needs to be, right?

Reply:
True, but these values are arbitrary, like the `58` for margin, other values are based on `$gutter` or have some logic behind them.

Reply:
It doesn't _have_ to be arbitrary, it could be based off something. Maybe even `($gutter * 2)`; or some other new variable.

Reply:
@k80 i think if we add a variable there it could be mistaken for an integral non-editable part of the mixin though. an arbitrary value might look a bit messy, but at least it indicates a value you can edit? maybe we could have warnings in there to update those values and just have some kind of obvious placeholder value?

Reply:
`123456789px` would be obvious… and totally break the design.